external-builds/pytorch/skip_tests/README.md

```diff
@@ -0,0 +1,90 @@
+# Skipping pytorch tests
+
+## Introduction
+
+This tooling allows to narrow down pytorch tests by skipping explicitely tests.
+Either in general, or additioanlly filtered by `amdgpu_family` and/or `pytorch_version`.
+
+By default, we are trying to follow the recommended [PyTorch test suite](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/3rd-party/pytorch-install.html#testing-the-pytorch-installation).
+Below you find the extract of it (1st Oct, 2025).
+
+```
+PYTORCH_TEST_WITH_ROCM=1 python3 test/run_test.py --verbose \
+--include test_nn test_torch test_cuda test_ops \
+test_unary_ufuncs test_binary_ufuncs test_autograd
+```
+
+Note:
+1st Oct 2025: Missing is `test/test_ops.py` and `test/inductor/test_torchinductor.py` due to issues in Triton.
+
+However, the filtering using `PYTORCH_TEST_WITH_ROCM=1` is not fully reflecting our failures. As such, this tooling will provide a more fine-grained filtering mechanism in our control to skip additional tests.
+
+Independent of this tooling, it is _always_ welcome to _get those changes upstream_! :)
```

Comment:
:D yes

Relating to my other comment, I think some iteration here in this repository is probably fine, but let's keep a careful eye on this to make sure that 

1. It's not adding too much complexity to ROCm development processes
2. We're staying aligned with upstream and are not doing significant engineering out of tree that is hidden from developers working directly in pytorch itself